src/MagnumPlugins/AnyImageImporter/AnyImageImporter.h

```diff
@@ -66,6 +66,7 @@ of the `Magnum` package and link to the `Magnum::AnyImageImporter` target. See
 
 Supported formats:
 
+-   Basis Universal (`*.basis`), loaded with any plugin that provides `BasisImporter`
```

Comment:
Should say `with @ref BasisImporter or any other plugin that provides it` instead ... for `BmpImporter` etc there's no dedicated plugin so nothing to link to, but here (and in case of `PngImporter`, `DdsImporter`, ...) it is.

Same for the `BasisImageConverter` (and then I can't merge that part until the converter PR is merged otherwise the docs won't build cleanly).

Reply:
I'm fine with getting this ready to merge and then you can merge it later with the converter 👍